WeakAurasOptions/AuthorOptions.lua

```diff
@@ -1233,9 +1259,10 @@ local function addControlsForOption(authorOptions, args, data, order, i)
   args["option" .. i .. "header"] = {
     type = "description",
     width = WeakAuras.doubleWidth - 0.75,
-    name = nameHead(data, option, option.name
+    name = nameHead(data, option, option.name -- TODO: find a better way
```

Comment:
is this something you want to solve before merging?

Reply:
I'm fine with leaving it for now. It's a low priority thing (i.e. it works), I just don't particularly enjoy editing it.

The TODO is so that I don't miss it. I've changed the comment to be more specific.